documentation/database.html

```diff
@@ -1,324 +1,451 @@
+
+            <h3>Setup Instructions</h3>
+
+            <ol>
+                <li>
+                    Create a new database using Enterprise Manager if you do not already have a database you want to
+                    use. You may wish to name the database "openfire".
+                </li>
+                <li>
+                    Create a user to access the database if you do not already have one you'd like to use. Consider
+                    making the default database for the user be the one you created in step 1.
+                </li>
+                <li>
+                    Proceed to the Openfire setup tool and use the appropriate JDBC settings when prompted.
+                </li>
+            </ol>
+
+        </section>
+
+    </section>
+
+    <section id="postgres">
+
+        <h2>PostgreSQL</h2>
+
+        <section id="postgres-jdbc-drivers">
+
+            <h3>JDBC Drivers</h3>
+
+            <p>
+                The JDBC driver for PostgreSQL is bundled with Openfire, so you do not need to download and install a
+                separate driver.
+            </p>
+            <p>
+                In the Openfire web-based setup tool, use the following values:
+            </p>
+            <ul style="list-style: none">
+                <li><span class="short-tab">driver:</span> <code>org.postgresql.Driver</code></li>
+                <li><span class="short-tab">server:</span> <code>jdbc:postgresql://HOSTNAME:PORTNUMBER/DATABASENAME</code></li>
+            </ul>
+            <p>
+                If left out, host defaults to localhost (not 127.0.0.1) for HOSTNAME and 5432 for PORT-NUMBER.
+            </p>
+
+        </section>
+
+        <section id="postgres-setup-instruction">
+
+            <h3>Setup Instructions</h3>
+
+            <p>
+                A general best practice is to create a PostgreSQL user dedicated for Openfire's use. This user need not
+                be a super-user, but should own the dedicated database used by Openfire.
+            </p>
+            <ol>
+                <li>
+                    Create a user for Openfire to connect as:
+
+                    <ul><li style="list-style: none">
+                        <code>createuser -P openfireuser</code>
+                    </li></ul>
+                </li>
+                <li>
+                    Create the database owned by this openfire user. For example, using the PostgreSQL
+                    <code>&quot;createdb&quot;</code> utility:
+                    <ul>
+                        <li style="list-style: none">
+                            <code>createdb -E UNICODE -O openfireuser openfire</code>
+                        </li>
+                    </ul>
+                </li>
+                <li>
+                    Proceed with Openfire install setup via the admin console, the setup process with install the schema.
+                </li>
+            </ol>
+
+        </section>
```

Comment:
Worth adding a useful link where we can, like with mysql?

```suggestion
                <li>
                    Proceed with Openfire install setup via the admin console, the setup process with install the schema.
                </li>
            </ol>

        </section>
         <section id="postgres-further-help">

            <h3>Further Help</h3>

            <p>
                If you need help setting up Postgres, refer to the following site:
            </p>
            <ul>
                <li><a href=["https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-reference-configuration-properties.html](https://jdbc.postgresql.org/documentation/use/#connecting-to-the-database)">Postgres Configuration and Connection documentation</a></li>
            </ul>

        </section>
```